app/views/OnboardingView/styles.js

```diff
@@ -1,41 +0,0 @@
-import { StyleSheet } from 'react-native';
-
-import { moderateScale, verticalScale } from '../../utils/scaling';
-import { isTablet } from '../../utils/deviceInfo';
-import sharedStyles from '../Styles';
-
-export default StyleSheet.create({
-	onboarding: {
-		alignSelf: 'center',
-		marginTop: isTablet ? 0 : verticalScale(116),
-		marginBottom: verticalScale(50),
-		maxHeight: verticalScale(150),
```

Comment:
If we stick to scaling, remember to do it on render instead.
There was an issue when the app was started on landscape and styles calculated on `styles.js`.